app/views/api/v2/hosts/base.json.rabl

```diff
@@ -1,3 +1,3 @@
 object @host
 
-attributes :name, :id
+attributes :id, :name
```

Comment:
Could you add displayname here so plugins could use it directly?
```
node :displayname do |host|
  host.to_label 
end
```

Reply:
Sure thing.